### PR TITLE
Replace setup.py by pyproject.toml

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by merwok** [comment]
That’s probably up to the build backend

File: MANIFEST.in [hunk 1]
```diff
@@ -1,5 +0,0 @@
-include *.txt *.py *.rst setup.cfg
```
Comment (Natim) [inline MANIFEST.in:1]: Why did you remove this file?

```
global-exclude */__pycache__/*
global-exclude *.pyc
```

**Comment by claudep** [comment]
I think this is now ready. @Natim, is this OK for you?
